daemons/gptp/common/md_wireless.cpp

```diff
@@ -231,7 +240,7 @@ bool MediaDependentWirelessPort::processSync(uint16_t seq, bool grandmaster, lon
 				unsigned residence_time;
 				follow_up->setPreciseOriginTimestamp
 					(getClock()->getLastSyncOriginTime());
-				// Calculate residence time
+				// Calculate residence time  XXX : This looks suspicious as if residence time could be negative
```

Comment:
I’ll remove the comment, it may have made sense at one time, but not now.

From: Levi Pearson [mailto:notifications@github.com]
Sent: Friday, December 04, 2015 2:23 PM
To: AVnu/Open-AVB
Cc: Hall, Christopher S
Subject: Re: [Open-AVB] Added support for Intel wireless adapters on Windows (#324)


In daemons/gptp/common/md_wireless.cpp<https://github.com/AVnu/Open-AVB/pull/324#discussion_r46739792>:

> @@ -231,7 +240,7 @@ bool MediaDependentWirelessPort::processSync(uint16_t seq, bool grandmaster, lon

>                             unsigned residence_time;

>                             follow_up->setPreciseOriginTimestamp

>                                    (getClock()->getLastSyncOriginTime());

> -                           // Calculate residence time

> +                           // Calculate residence time  XXX : This looks suspicious as if residence time could be negative

Could you elaborate on your concern in this comment?

—
Reply to this email directly or view it on GitHub<https://github.com/AVnu/Open-AVB/pull/324/files#r46739792>.